partid/partid.go

```diff
@@ -61,23 +61,23 @@ var (
 
 // Text gives human readable names
 var Text = map[[16]byte]string{ // nolint:gochecknoglobals
-	EFI:             "EFI System Partition",
+	EFI:             "EFI",
 	LUKS:            "LUKS",
-	LinuxFS:         "Linux",
+	LinuxFS:         "Linux-FS",
 	LinuxRAID:       "RAID",
-	LinuxRootX86:    "Linux root (ia32)",
-	LinuxRootArm32:  "Linux root (arm32)",
-	LinuxRootX86_64: "Linux root (x86_64)",
-	LinuxRootArm64:  "Linux root (arm64)",
-	LinuxDMCrypt:    "dm-crypt",
-	LinuxReserved:   "Linux Reserved",
+	LinuxRootX86:    "Linux-root-x86",
+	LinuxRootArm32:  "Linux-root-arm32",
+	LinuxRootX86_64: "Linux-root-x86_64",
+	LinuxRootArm64:  "Linux-root-arm64",
+	LinuxDMCrypt:    "DM-crypt",
+	LinuxReserved:   "Linux-Reserved",
 	LinuxLVM:        "LVM",
-	LinuxBoot:       "Linux /boot",
-	LinuxSwap:       "Linux swap",
-	LinuxHome:       "Linux /home",
-	LinuxSrv:        "Linux /srv",
+	LinuxBoot:       "Linux-/boot",
```

Comment:
It looks like you're more interested in getting rid of spaces than shortening.  Is something breaking because of spaces?

Reply:
well, i wanted it to be parsible, to have some delimiter.  that obviousl goes against what I just said above, which is that its not for parsing.  

i dont know. i dont like spaces if they're not needed.  this is s semi-middle ground.